content/_includes/home/contributers-and-governance.html

```diff
@@ -55,7 +55,7 @@ <h3 id="Individual_Contributors" class="section">Individual Contributors <a id="
 *   Janko Simonović [janko.simonovic@sbgenomics.com](mailto:janko.simonovic@sbgenomics.com)
 *   Stian Soiland-Reyes, University of Manchester; [https://orcid.org/0000-0001-9842-9718](https://orcid.org/0000-0001-9842-9718)
 *   Luka Stojanovic [luka.stojanovic@sbgenomics.com](mailto:luka.stojanovic@sbgenomics.com)
-*   Tomoya Tanjo, NII; [https://orcid.org/0000-0002-4421-9659](https://orcid.org/0000-0002-4421-9659)
+*   Tomoya Tanjo, NIG; [https://orcid.org/0000-0002-4421-9659](https://orcid.org/0000-0002-4421-9659)
```

Comment:
Feel free to add a link to https://www.nig.ac.jp/nig/ for those of us who are a bit ignorant :-P

Reply:
```suggestion
*   Tomoya Tanjo, [NIG](https://www.nig.ac.jp/nig/); [https://orcid.org/0000-0002-4421-9659](https://orcid.org/0000-0002-4421-9659)
```